50SAT/SPEX$50SAT.tex

```diff
@@ -0,0 +1,353 @@
+\end{itemize}
+The published designs also include an arduino-based radio receiver station. The team could develop this component as well, or preferably, leverage other teams and their projects to satisfy this requirement.
+Finally, the team will need to generate a presentation poster that could be used at ImagineRIT or other public SPEX events to highlight the project.
+
+\subsection{Milestones}
+\label{subsec:milestones}
+  % Be as detailed as you can, but it's okay if there are unknowns.
+  % At the very least, specify how many semester you expect the project to take until it reaches completion.
+If this project is conducted over a semester timeline, the team will need to expediently cover the required areas defined below.
+
+This project will take place over three technical domains: electrical, mechanical, and software.
+
+For the electrical systems, the team will need to review the existing PCB designs, primarily to determine whether it is feasible to manufacture and populate them on campus.
+Also the team will need to determine if all components required are still available, and if not, find suitable replacements and make the necessary design changes.
+PCB assembly will be the system with the longest lead time and should be prioritized first to ensure no bottlenecks on the critical path.
+Once assembled, each will need to be tested, and integrated within the system structure.
+
+For the mechanical systems, the team should evaluate the feasibility of manufacturing the frame on-campus.
+Any tweaks to the internal structure and placement of internal components should be locked down as early as possible.
+The mechanical team will also be responsible for testing the frame and whole assembly for verification against thermal and vibration requirements.
+
+For the onboard software, the team should review the existing code, and try to demonstrate functionality on similar hardware to prevent dependency on the electrical team.
+Once the primary electronics are ready, the software should be tested for functionality on the system hardware.
+
+The timeline below is preliminary:
+
+\begin{table}
+\centering
+ \begin{tabular}{|c | c | c|}
+ \hline
+ Phase & Task & Duration \\ [0.5ex]
+ \hline\hline
+ 1 & Review the existing \$50SAT designs and materials &  no more than 2 weeks \\
+ \hline
+ 2 & Subsystem development & 6 weeks \\
+ \hline
+ 2 & Order PCB design and/or assembly & 6 weeks \\
+ \hline
+ 2 & Review changes to mechanical strcuture and order materials & no more than 2 weeks \\
+ \hline
+ 2 & Testing of individual subsystems & 2 weeks \\
+ \hline
+ 3 & System Assembly & 1 week  \\
+ \hline
+ 4 & System testing & 2 weeks  \\
+ \hline
+ 5 & Generate documentation and delivery to SPEX & 1 week  \\
+ \hline
+\end{tabular}
+\label{tab:timeline}
+\end{table}
+
+\begin{itemize}
+  \item Review the existing \$50SAT designs and materials - no more than 2 weeks
```

Comment:
I think you're using the wrong dash here. It should be an _en_ dash or _em_ dash instead of a hyphen (I forget which). 

LaTeX makes one automatically if you use `--`. 

If you haven't already, install [linter-chktex](https://atom.io/packages/linter-chktex) or a LaTeX linter, it suggests corrections to style errors like this.